chrome/content/zotero/xpcom/utilities_internal.js

```diff
@@ -1735,6 +1735,28 @@ Zotero.Utilities.Internal = {
 	},
 	
 	
+	showInLibrary: async function (itemID) {
```

Comment:
This should take a `Zotero.DataObject`, not an `itemID`. Can throw an error for types other than `Zotero.Item` now if we don't want to support them yet.